geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/collection/CloseableIterator.scala

```diff
@@ -70,6 +70,9 @@ trait CloseableIterator[+A] extends Iterator[A] with Closeable {
 
   override def filter(p: A => Boolean): CloseableIterator[A] = CloseableIterator(super.filter(p), self.close())
 
+  override def collect[B](pf: PartialFunction[A, B]): CloseableIterator[B] =
```

Comment:
Should we extend the CI unit test to cover the new method?

Reply:
wilco